WordPress/gutenberg
# Add 'View site' action to 'Site updated' snackbar

<img width="443" alt="Screenshot 2023-07-17 at 15 56 46" src="https://github.com/WordPress/gutenberg/assets/846565/84eb0970-014c-4172-97fd-3c4dfcf283ac">


## What?
Add a 'View site' action to the 'Site updated' snackbar in the site editor.

## Why?
This seems like a moment where the user may want to view their changes on the frontend.

## Testing Instructions
1. Open the site editor.
2. Change the site in any way (edit a page, template, styles, etc).
3. Save changes
4. Observe the "View site" link in the resulting Snackbar
5. Click "View site" and ensure the link works
